.gitignore

```diff
@@ -1,2 +1,154 @@
 *.feather
 .venv
+# Byte-compiled / optimized / DLL files
```

Comment:
Would love to keep this file more manageable by limiting only to files that are actually produced. If I'm not mistaken, you could simplify this file by limiting to a few compiled assets and some distribution-related files:

```
*.feather
.venv

# Byte-compiled / optimized / DLL files
__pycache__/
*.py[cod]

# Distribution/packaging
build/
dist/
*.egg-info
```